### ISSUE TITLE
More Blame View Errors

### ISSUE THREAD

**Issue by nocarryr** [takezoe/gitbucket#781]
The error found and corrected in #773 seems to affect some configurations differently.

I have the same issue, but only after the fix was implemented.  It seems the original would have worked properly, but I just upgraded from 2.7 to 3.3.  Here's what I found in Chrome DevTools:

```
Uncaught TypeError: Cannot read property '1' of null

>> repo = $(".blame-action").data("repository")
"http://myhiddensubdomain.myhiddendomainname.com/nocarrier/amx_modules"

>> location.pathname
"/nocarrier/amx_modules/blob/master/SymnetIP.axs"

>> location.href
"http://myhiddensubdomain.myhiddendomainname.com/nocarrier/amx_modules/blob/master/SymnetIP.axs"

>> /^\/(blame|blob)(\/.*)$/.exec(location.pathname.substring(repo.length))
null

>> /^\/(blame|blob)(\/.*)$/.exec(location.href.substring(repo.length))
["/blob/master/SymnetIP.axs", "blob", "/master/SymnetIP.axs"]
```

It would seem as though location.href would have worked in my case.  Could it possibly be due to my subdomain configuration?

**Comment by pibi** [comment]
It works if you set the Base URL. We use gitbucket from different domains, so we can't set the Base URL and we are affected byt his bug too.

**Comment by takezoe** [comment]
This bug will be solved by #864.
